storage-engine/titan-configuration.md

```diff
@@ -39,9 +39,13 @@ Titan 对 RocksDB 兼容，也就是说，使用 RocksDB 存储引擎的现有 T
     enabled = true
     ```
 
```

Comment:
```suggestion

+ 方法三：编辑 TiDB-Operator 的 ${cluster_name}/tidb-cluster.yaml 配置文件，编辑示例如下：
  
    {{< copyable "" >}}

    ``` yaml
      tikv:
        ## Base image of the component
        baseImage: pingcap/tikv

        ## tikv-server configuration
        ## Ref: https://docs.pingcap.com/tidb/stable/tikv-configuration-file
        config: |
          log-level = "info"      

          [rocksdb]
            [rocksdb.titan]
              enabled = true
    ```

    应用配置时默认会触发在线滚动重启 TiDB 集群让配置生效：

     {{< copyable "shell-regular" >}}

    ```shell
    kubectl apply -f ${cluster_name} -n ${namespace}
    ```   

    更多信息可参考[在 Kubernetes 中配置 TiDB 集群](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/configure-a-tidb-cluster)。

```